.travis.yml

```diff
@@ -16,6 +16,8 @@ matrix:
   include:
     - php: 5.6
       env: DEPENDENCIES='low'
+    - php: 5.6
+      env: DEPENDENCIES='low' SKIP_INTEROP='~@interop&&'
```

Comment:
IMO, that should not be a `DEPENDENCIES='low'` job.

Reply:
Why do you think so? `container-interop` is an abandon package, so that is test backward compatibility with the oldest supported versions.

Reply:
well, this job is not about testing `container-interop`

Reply:
Job? I don't understand. It seem like ... um ... environment variable. What do you suggest?

Reply:
I've found your description about that in other PR. I agree with you.